sdk/template/template/package.json

```diff
@@ -111,12 +106,9 @@
     "prettier": "^1.16.4",
     "rimraf": "^3.0.0",
     "rollup": "^1.16.3",
-    "rollup-plugin-shim": "^1.0.0",
-    "rollup-plugin-sourcemaps": "^0.4.2",
-    "rollup-plugin-terser": "^5.1.1",
-    "rollup-plugin-visualizer": "^4.0.4",
     "typescript": "~3.9.3",
-    "util": "^0.12.1"
+    "util": "^0.12.1",
+    "karma-typescript": "~5.0.3"
```

Comment:
is this needed? we do not have it in TA.

Reply:
This is a leftover from when I was playing with this plugin in the karma config, thanks for pointing this out!